Chumigos/Chumigos/Games/Avatar Game/Model/FeelingClassificator.swift

```diff
@@ -7,31 +7,37 @@
 //
 
 import Foundation
-import Vision
+import UIKit
 
 class FeelingClassificator{
     
-//    Image (Color 299 x 299)
-    
-    let model = logginhoFeelings2_0()
-    
-    init() {
+    /// This fuction is for classify an image.
```

Comment:
This function is used to classify an image
or
This function is for classifying an image